tests/fixtures/fixtures_jobcollection.py

```diff
@@ -1,23 +1,24 @@
 import pytest
 
 from ..context import JobCollection
+from .fixtures_globals import PROJECT_ID
 
 
 @pytest.fixture()
 def jobcollection_single_mock(auth_mock, job_mock):
-    return JobCollection(auth=auth_mock, project_id=auth_mock.project_id, jobs=[job_mock])
+    return JobCollection(auth=auth_mock, project_id=PROJECT_ID, jobs=[job_mock])
 
 
 @pytest.fixture()
 def jobcollection_multiple_mock(auth_mock, jobs_mock):
-    return JobCollection(auth=auth_mock, project_id=auth_mock.project_id, jobs=jobs_mock)
+    return JobCollection(auth=auth_mock, project_id=PROJECT_ID, jobs=jobs_mock)
 
 
 @pytest.fixture()
 def jobcollection_empty_mock(auth_mock):
-    return JobCollection(auth=auth_mock, project_id=auth_mock.project_id, jobs=[])
+    return JobCollection(auth=auth_mock, project_id=PROJECT_ID, jobs=[])
 
 
 @pytest.fixture()
-def jobcollection_live(auth_live, jobs_live):
-    return JobCollection(auth=auth_live, project_id=auth_live.project_id, jobs=jobs_live)
+def jobcollection_live(auth_live, project_id_live, jobs_live):
```

Comment:
I thought eventually we want to get rid of all live tests.
Why are we refactoring live tests?
Am I missing something?

Reply:
We will get rid of them, since I don't know how they are used now I didn't feel safe just by dropping them, hence the refactoring